lib/wrapbox/configuration.rb

```diff
@@ -68,6 +69,7 @@ def self.load_config(config)
         config["enable_ecs_managed_tags"],
         config["propagate_tags"],
         config["launch_instances"]&.deep_symbolize_keys,
+        config["enable_execute_command"] || false
```

Comment:
Can you make the default value nil? The default value `false` forces users to upgrade the SDK that support `enable_execute_command`.

Reply:
Of course, I just updated.